lib/playground.dart

```diff
@@ -181,11 +181,12 @@ class Playground implements GistContainer, GistController {
     // Don't auto-run if we're re-loading some unsaved edits; the gist might
     // have halting issues (#384).
     bool loadedFromSaved = false;
-    if (querySelector('#dart-code') != null) {
+    Uri url = Uri.parse(window.location.toString());
+    if (url.hasQuery && url.queryParameters['dart'] != null) {
       Gist blankGist = createSampleGist();
-      blankGist.getFile('main.dart').content = Uri.decodeQueryComponent(querySelector('#dart-code').innerHtml);
-      blankGist.getFile('index.html').content = Uri.decodeQueryComponent(querySelector('#html-code').innerHtml);
-      blankGist.getFile('styles.css').content = Uri.decodeQueryComponent(querySelector('#css-code').innerHtml);
+      blankGist.getFile('main.dart').content = url.queryParameters['dart'];
+      blankGist.getFile('index.html').content = url.queryParameters['html'];
```

Comment:
Can you manually verify that not having an html or css parameter on the url will not cause issues (that we won't have problems if we're assigning `null` to `.content`)?